src/scripts/main.js

```diff
@@ -1,3 +1,19 @@
 'use strict';
 
-// write your code here
+const table = document.querySelector('table');
+
+const trs = table.rows;
+
+function addColumn(lines) {
+  const rows = [...lines];
+
+  rows.forEach((row) => {
+    const tr = row.cells;
+    const secondTr = tr[1].cloneNode(true);
+
+    row.append(secondTr);
+    row.append(tr[4]);
```

Comment:
Appending `tr[4]` assumes that there are at least five cells in each row. This can lead to errors if any row has fewer than five cells. Ensure that all rows have the expected number of cells before accessing `tr[4]`.